### PR TITLE
Enhancement/priority calender

### PR DESCRIPTION
**Pull request by joshinehajoshi**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,5 +1,5 @@
 REACT_APP_CUSTOM_NODE_ENV = "development"
-REACT_APP_API_ENDPOINT =  http://localhost:4000/api/v1
```
Comment (mak9456) [inline .env:2]: Remove this change, not required

File: src/components/TaskList/AddTask.js [hunk 2]
```diff
@@ -869,7 +868,8 @@ const AddTask = (props) => {
               changeToDashboard();
             }}
           >
-            Save
```
Comment (mak9456) [inline src/components/TaskList/AddTask.js:872]: Why we have removed 'Save'

File: src/components/TaskList/AddTask.js [hunk 3]
```diff
@@ -254,7 +254,6 @@ const AddTask = (props) => {
         );
       }
     }
-    console.log("filtered tasks", props.task_lists_task)
```
Comment (mak9456) [inline src/components/TaskList/AddTask.js:257]: Remove console logs

File: src/components/dashboard/AddTaskModal.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import React, { Component } from "react";
```
Comment (mak9456) [inline src/components/dashboard/AddTaskModal.js:1]: Why have we removed Component here

File: src/components/dashboard/DashboardEvent.js [hunk 5]
```diff
@@ -23,6 +23,7 @@ import ClockIcon from "../../assets/images/hourglasss.gif";
 
 import PauseIcon from "../../assets/images/hourglasss.gif";
 import PlayIcon from "../../assets/images/hourglass1.png";
+import { NEWPRIORITIES } from '../../utils/Constants';
```
Comment (mak9456) [inline src/components/dashboard/DashboardEvent.js:26]: You have imported this but have you used this?

File: src/utils/Constants.js [hunk 6]
```diff
@@ -58,6 +58,14 @@ export const PRIORITIES = [
     label: "no priority",
   },
 ];
+
+export const NEWPRIORITIES = {
+  'low': {'label':'low', 'color_code':'#00A031'},
```
Comment (mak9456) [inline src/utils/Constants.js]: Make 'label'  to label.
'color_code' to color_code.

Essentially remove them as string form

File: prac.js [hunk 7]
```diff
@@ -0,0 +1,33 @@
+<button
+type="button"
```
Comment (mak9456) [inline prac.js]: This File is not required remove it

File: src/components/dashboard/AddTaskModal.js (hunk 4 above)
Comment (Sunildn) [reply src/components/dashboard/AddTaskModal.js:1]: while createing class component React.component is used

**Review by mak9456 — APPROVED** [review]

**Pull request merged**